README.md

```diff
@@ -139,7 +139,9 @@ Community Maintained Cars and Features
 | Holden    | Astra 2017<sup>1</sup>        | Adaptive Cruise   | openpilot        | 0mph               | 7mph         |
 | Hyundai   | Elantra 2017-19               | SCC + LKAS        | Stock            | 19mph              | 34mph        |
 | Hyundai   | Genesis 2015-16               | SCC + LKAS        | Stock            | 19mph              | 37mph        |
-| Hyundai   | Ioniq Electric 2019-20        | SCC + LKAS        | Stock            | 0mph               | 32mph        |
+| Hyundai   | Ioniq Electric 2020           | SCC + LKAS        | Stock            | 0mph               | 0mph         |
+| Hyundai   | Ioniq Hybrid                  | SCC + LKAS        | Stock            | 22mph              | 32mph        |
+| Hyundai   | Ioniq Electric Limited 2019   | SCC + LKAS        | Stock            | 0mph               | 32mph        |
```

Comment:
@VirtuallyChris can you check this and add the year for the hybrid?

Reply:
According to XPS, we don't support any Hybrids yet.

> hybrids have different signals packed into the same message EVs have different info. so this puts all hybrids non compatible with stock op https://github.com/xps-genesis/openpilot/blob/bfd334b7d8a9eb7ccdd5ef76e9beb23a3cc282e6/opendbc/hyundai_kia_generic_hybrid.dbc#L1515 vs https://github.com/xps-genesis/openpilot/blob/bfd334b7d8a9eb7ccdd5ef76e9beb23a3cc282e6/opendbc/hyundai_kia_generic.dbc#L1515

Let's remove the Ioniq Hybrid and just keep the two electric models. The 'IONIQ_EV_LTD' should be renamed to 'IONIQ_EV' since the trim is irrelevant and we can keep the new one as 'IONIQ_EV_2020'. Also we should remove 'Limited' in the readme since this is also irrelevant.

Both of these have SCC standard, and both have stop and go. The 2019 does cut steering at 32mph but the 2020 can steer down to 0, so that's all correct.